movieapp/src/models/types.ts

```diff
@@ -22,17 +24,26 @@ export type Movie = {
   [id: string]: string;
   title: string;
   description: string;
+  fullDescription: string;
+};
```

Comment:
Movie is a model and it's valid to store it here

props interfaces should be near components/containers/pages

Reply:
fixed